Alcatraz/Helpers/ATZShell.m

```diff
@@ -77,8 +77,11 @@ - (void)setUpTerminationHandlerForTask:(NSTask *)task completion:(void(^)(NSStri
             if (task.terminationStatus == 0) {
                 completion(output, nil);
             } else {
-                NSString* reason = [NSString stringWithFormat:@"Task exited with status %d", task.terminationStatus];
-                completion(output, [NSError errorWithDomain:reason code:666 userInfo:@{ NSLocalizedDescriptionKey: reason }]);
+                const NSUInteger maxLength = 500;
+                NSString *truncatedOutput = [[output substringToIndex:MIN(maxLength, output.length)] stringByAppendingString:output.length > maxLength ? @"…" : @""];
```

Comment:
I'm guessing we need to truncate the output to avoid displaying a humongous modal dialog. Still, I feel that we should log the full output in case we need to debug the error.